lib/code_corps/policy/policy.ex

```diff
@@ -31,6 +31,9 @@ defmodule CodeCorps.Policy do
   defp can?(%User{} = user, :create, %Organization{}, %{}), do: Policy.Organization.create?(user)
   defp can?(%User{} = user, :update, %Organization{} = organization, %{}), do: Policy.Organization.update?(user, organization)  
   defp can?(%User{} = current_user, :update, %User{} = user, %{}), do: Policy.User.update?(user, current_user)
```

Comment:
Also not in scope of this PR but it's odd to me seeing `current_user` as the first argument only in this case. That seems backwards, no?

Reply:
oh...this was b/c it was for the `user` update action...so to differentiate the variable names.  Would you like it renamed?  Maybe `user` as first arg, then `user_struct` as second arg?

Reply:
Yeah, what I had meant is that the first param in the policy is `user` and the second is `current_user`. That actually seems wrong. And, in fact, in the other policies the `user` _should be_ `current_user`! At least, this is how I currently understand it. Is that the right reading of the situation?

Reply:
Yeah that might be a good idea.  Change user to `current_user` as the pattern matched first arg.

Reply:
@snewcomer, @JoshSmith  We can actually do that right now, considering how the policy is implemented.

```Elixir
defp can?(%User{} = current_user, :update, %User{} = user, %{}), do: Policy.User.update?(current_user, user)
```

If you look at the policy itself, it just checks if the two records have the same ids, so we can reorder them without consequences at the moment.